control-plane/pkg/core/config/utils_test.go

```diff
@@ -104,25 +104,48 @@ func TestBackoffDelayFromString(t *testing.T) {
 	tests := []struct {
 		name         string
 		backoffDelay *string
-		defaultDelay string
-		want         string
+		defaultDelay uint64
+		want         uint64
+		wantError    bool
 	}{
 		{
```

Comment:
Another test case with microseconds? To check if something breaks or the result is properly 0 milliseconds :smile:

Reply:
Done https://github.com/knative-sandbox/eventing-kafka-broker/pull/284/commits/5b51863d2049ab020608df9a31cae32e5ac9f140